src/lib/components/Banner.svelte

```diff
@@ -0,0 +1,152 @@
+<script lang="ts">
+	import { createEventDispatcher } from 'svelte';
+	import Button from './Button.svelte';
+
+	export let level: 'info' | 'warning' | 'caution' = 'info';
+
+	const dispatch = createEventDispatcher();
+	let banner: Element;
+
+	const removeElement = (e: AnimationEvent) => {
+		if (e.animationName.includes('swipeUp')) (e.target as Element).remove();
```

Comment:
Why is the check needed or why is the element being removed? Once the banner is outside of the view (top -100%) it isn't visible anyway

Reply:
the check is needed because the animationend event fires for every animation, even the one that makes the banner appear

the element is removed because, since the user dismisses it, they don't want to see it again

Reply:
If the user dismisses it the animation hides it as it is off screen, why would you need to additionally remove it?

Reply:
why would it need to be kept in the DOM anyways?

Reply:
It doesn't need, but it also doesn't need to be removed.

Reply:
I honestly think it does need to. imagine you have 1000 different banners, why would you leave 1000 elements in the DOM if they're not visible?

Reply:
Theres only one banner and even if it were 10 or 100 it wouldn't matter

Reply:
resolved in b73eb05